packages/ozone/src/auth-verifier.ts

```diff
@@ -82,14 +85,13 @@ export class AuthVerifier {
 
   moderator = async (reqCtx: ReqCtx): Promise<ModeratorOutput> => {
     const creds = await this.standard(reqCtx)
-    if (!creds.credentials.isTriage) {
+    if (!creds.credentials.isTriage && !creds.credentials.isVerifier) {
       throw new AuthRequiredError('not a moderator account')
     }
```

Comment:
This previously was the check that the user had at least triage access, and everywhere else I expect it was implicit.  Do we need to add explicit `isTriage` checks to any endpoints to restrict members who are verifiers but not triage?

Reply:
I don't believe there are any actions that triage mods can perform that we'd want to block verifiers from doing but I'll get this confirmed.